.github/workflows/codeql.yml

```diff
@@ -24,7 +24,7 @@ jobs:
       fail-fast: false
       matrix: 
         # Autobuild each of these seperate maven projects
-        working-directory: ['invoker', 'functions-framework-api', 'function-maven-plugin']
+        working-directory: ['functions-framework-api', 'invoker', 'function-maven-plugin']
```

Comment:
the order needed to be changed?

Reply:
reverted this change, this works in any order.